webapp/web/dashboard.css

```
@@ -1,91 +1,233 @@
 }
 
-.active-packages {}
+.project-audience-action:hover {
+  cursor: pointer;
+}
+
+.project-audience-action__text {
+  white-space: nowrap;
+  font-size: 1.2em;
+}
+
+.project-audience-action__icon {
+  margin-left: 10px;
+  font-size: 1.4em;
+  line-height: 25px;
+}
+
+.project-audience-content {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid #cacaca;
+  padding: 20px;
+}
+
+.project-audience-content-item {
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 1.3em;
+  margin-bottom: 20px;
+}
 
-.active-packages__package {
-    border-style: solid;
-    border-width: 1px;
-    border-radius: .5rem;
-    border-color: rgba(0, 0, 0, .1);
-    padding: 6px;
-    margin-bottom: 20px;
-    width: 80%
+.project-audience-content-item__text {
+  flex-basis: 100%;
 }
 
-.active-packages__package-actions {
-    margin-bottom: 10px;
+.project-audience-content-item__description {
+  font-weight: 400;
 }
 
-.active-packages__package-action {
+.project-audience-content-item__action-link {
+  margin-left: 20px;
+  text-decoration: none;
+  color: #499398;
+}
+
+
+.package-group {
+  display: flex;
+  flex-direction: column;
+  margin-top: 50px;
+}
+
+.group-title {
+  color: #b3b3b3;
+}
+
+.package {
+    display: grid;
+    grid-template-columns: auto auto;
+    border: 2px solid #cacaca;
+    border-bottom-style: none;
     padding: 11px;
 }
 
-.available-packages {
-    display: flex;
-    flex-direction: column;
-    margin-top: 50px;
+.package--dotted {
+  border-style: dotted;
+  border-bottom-style: none;
 }
 
-.available-packages__package {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    border-style: solid;
-    border-width: 1px;
-    border-radius: .5rem;
-    border-color: rgba(0, 0, 0, .1);
-    padding: 6px;
-    margin-bottom: 20px;
-}
-
-.available-packages__add-package {
-    border-style: solid;
-    border-width: 1px;
-    border-width: .125rem;
-    align-self: center;
-    width: 25%;
-    text-align: center;
-    height: 60px;
-    margin: auto;
-    line-height: 60px;
+.package:last-child {
+  border-bottom-style: solid;
 }
 
-.available-packages__add-package:hover {
-    cursor: pointer;
+.package--dotted:last-child {
+  border-bottom-style: dotted;
+}
+
+.active-package-main-container {
+  align-self: center;
+}
+
+.active-package-title {
+  display: flex;
+  align-self: center;
+}
+
+.active-package-title__text {
+  font-size: 1.1em;
+  margin: 0px;
 }
 
-.available-packages__add-package-link {
-    color: #000000;
+.active-package-title__icon {
+  width: 16px;
+  height: 16px;
+  margin-left: 10px;
+}
+
+.active-package-actions {
+  margin: 10px 0px;
+}
+
+.active-package-actions__action-link {
+  padding: 10px 10px 10px 0px;
+  color: #499398;
+  text-decoration: none;
+}
+
+.active-package-chart {
+  justify-self: flex-end;
+}
+
+.active-package-chart__data {}
+
+.active-package-chart__description {
+  margin: 0px;
+}
+
+.available-package-name {
+  font-size: 1.1em;
```

let's make this bolder, it looks a bit too much like the rest of the text in the package box:
```suggestion
  font-size: 1.1em;
  font-weight: 600;
```